dockerfiles/Dockerfile_rstudio_4.1.0

```diff
@@ -6,7 +6,7 @@ LABEL org.opencontainers.image.licenses="GPL-2.0-or-later" \
       org.opencontainers.image.authors="Carl Boettiger <cboettig@ropensci.org>"
 
 ENV S6_VERSION=v2.1.0.2
-ENV RSTUDIO_VERSION=latest
+ENV RSTUDIO_VERSION=1.4.1106
```

Comment:
I think this should still be `latest`, in line with current convention (i.e. R packages are not frozen on 4.1.0 yet either, but are always the "latest" from CRAN, and 4.1.0 docker tag is currently an alias for `latest`, so it makes sense that it's versions are likewise too.  

Maybe that doesn't matter now that the script will effectively automatically update it anyhow?

Reply:
I also wondered which would be better, but I think I can keep up with version updates as it is, because if I use GitHub Actions to detect daily updates, you can reflect version upgrades immediately (the things you have to is only approve bot's PR).
Since the `docker build` action is triggered when the Dockerfile is updated, writing version number approach may be a more appropriate and convenient way.

Reply:
Today, the automatic execution of the script informed me that a new version of RStudioServer was available for download. https://github.com/eitsupi/r-ver/pull/67/files

Reply:
Agree, with automatic updating via the CI this sounds fine.